lux/interestingness/interestingness.py

```diff
@@ -363,3 +364,9 @@ def monotonicity(vis: Vis, attr_specs: list, ignore_identity: bool = True) -> in
         return -1
     else:
         return score
+
+
+def n_distinct(vis: Vis, dimension_lst: list, measure_lst: list) -> int:
```

Comment:
What is the goal of the `n_distinct`? I didn't quite understand this and was wondering if we should just assign dummy scores, since it is unlikely that a dataset would have too many geo attributes. Similar to what we do in `generalize.py`.

Reply:
`n_distinct` computes how many unique values there are for a geographical data type. For example, if a dataset displayed earthquake magnitudes across 48 states and 3 countries, `n_distinct` would return 48 and 3 respectively, meaning the states choropleth would be displayed first, and the countries choropleth second.

Reply:
Ah ok, this makes sense!